packages/back-end/src/services/queueing.ts

```diff
@@ -5,6 +5,7 @@ let agendaInstance: Agenda;
 
 export const getAgendaInstance = (): Agenda => {
   if (!agendaInstance) {
+    // @ts-expect-error - Type instantiation is excessively deep and possibly infinite.ts(2589)
```

Comment:
I don't know what's going on here but Agenda config appears to be accumulating these `@ts-expect-error` lines. Hopefully they upgrade to the latest mongodb soon to hopefully clean up this call site (https://github.com/agenda/agenda/issues/1525).